src/test/java/no/unit/nva/database/DatabaseServiceTest.java

```diff
@@ -110,7 +112,8 @@ public void addRoleThrowsConflictExceptionWhenTryingToSaveAlreadyExistingUser()
     @Test
     public void databaseServiceReturnsNonEmptyUserWhenUsernameExistsInDatabase()
         throws InvalidEntryInternalException, ConflictException, InvalidInputException, NotFoundException {
-        UserDto insertedUser = createSampleUserAndAddUserToDb(SOME_USERNAME, SOME_INSTITUTION, SOME_ROLE);
+        UserDto insertedUser = createSampleUserAndAddUserToDb(SOME_USERNAME, SOME_GIVEN_NAME, SOME_FAMILY_NAME,
```

Comment:
SomeGivenName and SomeFamliyName could be inlined and removed from the parameter list.